examples/gateway-bluegreen/03-deploy.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: apps/v1
```

Comment:
Thanks for this @izturn! 

One thought on organization - for `HTTPProxy`, most example configs like this live under `examples/example-workload/httpproxy`. I think it'd make sense to start a new subdirectory there, `examples/example-workload/gatewayapi`, and move this `HTTPRoute` file under there. As far as the CRDs, GatewayClass and Gateway, I'd be inclined to not create new copies of those, and perhaps just add a README.md in the `examples/example-workload/gatewayapi` documenting that the user is assumed to have gone through https://projectcontour.io/guides/gateway-api/ to set up Contour with Gateway API enabled

Reply:
done